ndcube/asdf/resources/schemas/ExtraCoords-0.1.0.yaml

```diff
@@ -0,0 +1,37 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/ExtraCoords-0.1.0"
+
+title:
+  Represents the ndcube ExtraCoords object
+
+description:
+  Represents the ndcube ExtraCoords object
+
+type: object
+properties:
+  wcs:
+    tag: "tag:stsci.edu:gwcs/wcs-1.*"
+  mapping:
+    type: array
+  lookup_tables:
+    type: array
+    items:
+        type: array
+        items:
+            - oneOf:
+              - type: number
+              - type: array
+            - oneOf:
+              - $ref: "asdf://sunpy.org/ndcube/schemas/QuantityTableCoordinate-0.1.0"
```

Comment:
Any reason to not use a `tag` validator here?

Reply:
Sorry, I couldn't find the reference documentation I mentioned earlier. I may have misread something about the $ref in the ASDF schema docs. I have replaced $ref with tags as we discussed.

Reply:
Thanks for looking and for pointing out the gap in the documentation. I opened an issue for asdf:
https://github.com/asdf-format/asdf/issues/1791
Feel free to add to it if I missed anything.